cmd/rokka/cli/massupload/worker.go

```diff
@@ -0,0 +1,62 @@
+package massupload
+
+import (
+	"github.com/rokka-io/rokka-go/rokka"
+	"fmt"
+	"os"
+	"path/filepath"
+	"math/rand"
+)
+
+func UploadWorker(client *rokka.Client, imageDetails chan ImageDetails, results chan UploadResult, quit chan struct{}, dryRun bool) {
+	// TODO: handle worker names?
+	id := fmt.Sprintf("%.2d", rand.Uint32() % 100)
```

Comment:
what's the id for? logging?

Reply:
Yes, I just wanted to output some info about the progress and which worker was doing what :)